cornice/service.py

```diff
@@ -241,10 +252,21 @@ def get_arguments(self, conf=None):
                 value.extend(to_list(conf.pop(arg)))
             arguments[arg] = value
 
+        if self.renderer == "simplejson":
+            # XXX: keep retrocompatibility
+            # when using 'simplejson' as renderer (default)
+            import simplejson
+
+            default_error_handler = json_error_handler(simplejson.dumps,
+                                                       {"use_decimal": True})
+        else:
+            default_error_handler = json_error_handler(
+                json.dumps, {"default": default_bytes_adapter})
```

Comment:
Could this be done in `.configure_default_renderer()` ?

Reply:
I agree. To be fair, I had a hard time understanding why is there a `renderer` object attached to the Service class, but then the same renderer is not for handling errors as well. 

Would it actually be better to find a way of using the `Service.renderer` also to perform the rendering of errors?